### PR TITLE
Remove version from configuration-as-code dependency

### PR DESCRIPTION
**Pull request by pascal-hofmann**
The plugin did not work with latest releases of configuration-as-code because it pinned to version 1.0.

### CODE REVIEW

**Comment by pascal-hofmann** [comment]
Hi @szandala, @jetersen!

Can I help to get this merged?

Cheers,
Pascal

File: pom.xml
Comment (jetersen) [inline pom.xml]: Why not the newest?

File: pom.xml
Comment (jetersen) [inline pom.xml]: This is a really old baseline, please consider the following.
https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/

File: pom.xml
Comment (jetersen) [inline pom.xml]: property is not needed just move it to where it is used.

**Review by jetersen — CHANGES_REQUESTED** [review]

**Comment by jetersen** [comment]
Did not notice the PR before you pinged but I would like to see these values updated at least. :)

**Comment by pascal-hofmann** [comment]
Hi @jetersen,
by now I completely forgot what I did here and why. 😅 

I tried to update my PR according to your comments. Can you have another look?

Cheers,
Pascal

**Comment by icep87** [comment]
@jetersen could you take a second look?

**Review by jetersen — APPROVED** [review]

**Comment by icep87** [comment]
@jetersen can you maybe trigger the tests once again and merge this pr?

**Comment by jetersen** [comment]
I think the Jenkins file needs updating 🤕

**Comment by icep87** [comment]
@pascal-hofmann any chance you could take a look?

**Comment by pascal-hofmann** [comment]
Sorry, this got buried under other things on my desk.

@jetersen Can you have another look? I assume my Jenkinsfile changes are not effective as I have not write access to the repo.

**Comment by pascal-hofmann** [comment]
I just had a look at the logs. It really does ignore my Jenkinsfile. 

```
[2024-01-05T16:31:40.257Z] Loading trusted files from base branch master at fd4f71096564a24f600e723b3263c4f973347c19 rather than 5a62240a3aa9c82f08b50b31ce960d6764389757
[2024-01-05T16:31:40.385Z] Obtained Jenkinsfile from fd4f71096564a24f600e723b3263c4f973347c19
[2024-01-05T16:31:40.607Z] ‘Jenkinsfile’ has been modified in an untrusted revision
```

**Comment by pascal-hofmann** [comment]
Hi @jetersen,
can you have another look?

Cheers
Pascal

**Comment by pascal-hofmann** [comment]
Hi @szandala, @jetersen!

I fixed the jenkinsfile, but it's not used in the build because it's untrusted. I guess a maintainer needs to approve the changes so it gets used?

Cheers,
Pascal

**Review by szandala — APPROVED** [review]

**Comment by szandala** [comment]
I will merge this, but revert if it fails

**Comment by pascal-hofmann** [comment]
Hmpf, this did not work. :(

It's a bit hard to test for me, without permission on the pipeline. Would be great if someone could have a look.

Thanks in advance,
Pascal

**Comment by szandala** [comment]
Tried here:
https://github.com/jenkinsci/configuration-as-code-groovy-plugin/pull/9
Will try to look :-/

**Pull request merged**
